radio/src/gui/480x272/radio_sdmanager.cpp

```diff
@@ -220,6 +224,11 @@ bool menuRadioSdManager(event_t _event)
           if (!strcasecmp(ext, SOUNDS_EXT)) {
             POPUP_MENU_ADD_ITEM(STR_PLAY_FILE);
           }
+          else if (!strcasecmp(ext, BMP_EXT) || !strcasecmp(ext, JPG_EXT) || !strcasecmp(ext, PNG_EXT)) {
```

Comment:
It's done inside sdListFiles() function, but not quite sure how it works to take it out and make it available elsewhere